packages/editor/src/lib/json-diff.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="ts">
+import { onDestroy } from 'svelte';
+import { MergeView } from '@codemirror/merge';
+
+import { createJSONExtensions, setLabel } from './codemirror';
+import EditorWrapper from './editor-wrapper.svelte';
+import classnames from 'classnames';
+
+/** Accessible label of the editor element. */
+export let labelPrefix: string;
+
+/** JSON Values before / after change */
+export let beforeValue: string;
+export let afterValue: string;
```

Comment:
This used to accept JSONObject and then do stringify + sort in here. However, it makes more sense to match the JSONEditor api and give users freedom to format the json.